src/familiar/marginalFamiliars.ts

```diff
@@ -116,8 +116,18 @@ function turnsNeededForFamiliar(
     case "none":
       return 0;
 
-    case "special":
-      return 0;
+    case "shrub":
+      return Math.ceil(estimatedTurns() / 100);
+
+    case "jellyfish":
+      return sum(
+        getAllJellyfishDrops().filter(
+          ({ expectedValue }) =>
+            outfitValue + familiarAbilityValue(familiar) + expectedValue >
+            totalFamiliarValue(baselineToCompareAgainst)
+        ),
+        ({ expectedTurns }) => expectedTurns
+      );
```

Comment:
Go back to using a case called `special`, and make a function that provides special handling for special familiars. Something like 
```
function getSpecialFamiliarLimit({
  familiar,
  baselineToCompareAgainst,
  outfitValue,
}: {
  familiar: Familiar;
  outfitValue: number;
  baselineToCompareAgainst: GeneralFamiliar & { outfitWeight: number; outfitValue: number };
}): number {
  switch (familiar) {
    case $familiar`Space Jellyfish`:
      return sum(
        getAllJellyfishDrops().filter(
          ({ expectedValue }) =>
            outfitValue + familiarAbilityValue(familiar) + expectedValue >
            totalFamiliarValue(baselineToCompareAgainst)
        ),
        ({ expectedTurns }) => expectedTurns
      );

    case $familiar`Crimbo Shrub`:
      return Math.ceil(estimatedTurns() / 100);

    default:
      return 0;
  }
}
```
which can live in this file at the bottom.

I don't want the types of limit to grow every time we get a weirdo, so instead we'll just put more cases in the special familiars.